migrator.go

```diff
@@ -114,10 +116,15 @@ func (m Migrator) CreateTable(models ...interface{}) error {
 				args = append(args, clause.Expr{SQL: index.Name}, indexOptions)
 			}
 			indexStr := strings.Join(indexSlice, ", ")
+			if len(indexSlice) > 0 {
+				indexStr = ", " + indexStr
+			}
 
 			// Step 4. Finally assemble CREATE TABLE ... SQL string
 			engineOpts := "MergeTree() ORDER BY tuple()"
```

Comment:
Should we read table option from `db.Get` to allow override the default value here??

Reply:
yes. I have a separate PR #7 : Added support for Table Engine. Please have a review on that PR also.

Reply:
So I think we can merge this one first. Second the PR #7 .

Next my plan is do:
-  Fix SQL `ALTER TABLE [ADD | COMMENT | MODIFY | DROP] COLUMN`
-  Fix SQL `ALTER TABLE [ADD | DROP] INDEX`
-  Support `GRANULARITY` in index option for `... INDEX ...` SQL
-  Integration testings. I think some tests need to have it's own for clickhouse. It has different behaviors than OLTP database.